app/modules/intelligence/agents_copy/chat_agents/crewai_rag_agent.py

```
@@ -0,0 +1,158 @@
+                    f"Input: {json.dumps(schema.get('example', {}), indent=2)}\n\n"
+                )
+        return tools_desc
+
+    def _create_task_description(
+        self,
+        task_config: TaskConfig,
+        ctx: ChatContext,
+    ) -> str:
+        """Create a task description from task configuration"""
+        if isinstance(ctx.node_ids, str):
+            node_ids = [ctx.node_ids]
```

Ruff (0.8.2)

Local variable `node_ids` is assigned to but never used

Remove assignment to unused variable `node_ids`

(F841)